project_event/models/task_category.py

```diff
@@ -18,3 +18,6 @@ class TaskCategory(models.Model):
         string='Sequence',
     )
     color = fields.Char()
+    is_default = fields.Boolean(
```

Comment:
We have to display the field in the form view (for admin to select the default value)

Reply:
Done